src/command_modules/azure-cli-role/azure/cli/command_modules/role/_help.py

```diff
@@ -193,6 +193,33 @@
 helps['role definition update'] = """
     type: command
     short-summary: Update a role definition.
+    parameters:
+        - name: --role-definition
+          type: string
+          short-summary: Description of a role as JSON, or a path to a file containing a JSON description.
+    examples:
+        - name: Create a role with read-only access to storage and network resources, and the ability to start or restart VMs.
+          text: |
+                az role definition create --role-definition { \\
+                    "Name": "Contoso On-call", \\
+                    "Description": "Perform VM actions and read storange and network information." \\
```

Comment:
Typo 'storange'?
Can you create another PR to fix?

Reply:
Well, I copy it from another command. I'll fix both places. Thanks!